registry/remote/utils.go

```diff
@@ -71,3 +71,14 @@ func withScopeHint(ctx context.Context, ref registry.Reference, actions ...strin
 	scope := auth.ScopeRepository(ref.Repository, actions...)
 	return auth.AppendScopes(ctx, scope)
 }
+
+// Helper function for TestRegistry_Repositories_WithLastParam
```

Comment:
If we really need this in multiple places, we should put it in the [internal](https://github.com/oras-project/oras-go/tree/main/internal) package.
Besides, the function doc must begin with the name of the function.